src/app/search/search.component.scss

```diff
@@ -1,20 +1,37 @@
-.wrapper-products {
-  display: flex;
-  flex-wrap: wrap;
-  flex-direction: row;
-  justify-content: space-around;
-  width: 100%;
-  margin: 0 auto;
-}
+$card-hover-bg: rgba(202, 199, 199, 0.11);
+$mat-select-bg: rgb(255, 255, 255);
 
-.t-100 {
-  margin-top: 100px;
+.my-card-content {
+  padding: 16px;
+}
+.my-card {
+  height: 350px;
+  width: 350px;
 }
-
 .card {
-  width: 16rem;
+  width: 100%;
+  margin: 10px;
+  max-width: 350px;
+  min-width: 270px;
+  height: 90%;
+  margin-bottom: 8px;
 }
-
 .card:hover {
-  background-color: rgba(128, 128, 128, 0.11);
+  background-color: $card-hover-bg;
+}
```

Comment:
The material guideline states, that hovering over cards on desktop elevates them:
https://material.io/components/cards#behavior